src/components/PupilComponents/QuizUtils/stemUtils.tsx

```diff
@@ -42,8 +44,10 @@ export const getStemImage = ({
         width={data.imageObject.width}
         height={data.imageObject.height}
         $minWidth={minWidth}
+        $maxWidth={scaled ? "all-spacing-21" : "all-spacing-0"}
         placeholder="oak"
-        sizes={getSizes(["100vw", 1200])}
+        sizes={getSizes(["100vw", scaled ? 3000 : 1200])}
```

Comment:
What are these hard-coded constants that you use throughout this PR. Can we get them into variables with descriptive names so that I understand what they are doing here. Perhaps wrap this all up in a util function so that each image instance has the same treatment ?

Reply:
everyones getting involved in the pr